sql/moz-fx-data-shared-prod/monitoring/schema_error_counts_v2/metadata.yaml

```
@@ -0,0 +1,11 @@
+---
+friendly_name: Schema Error Counts
+description: >
+  Counts the number of schema errors in the error stream per hour.
+owners:
+  - amiyaguchi@mozilla.com
+labels:
+  schedule: daily
+  incremental: false
```

Looks like it's incremental
```suggestion
  incremental: true
```
Not that it changes anything except for the label assigned to the table